### ISSUE TITLE
Add reporting contact

### ISSUE THREAD

**Issue by DOH-PXC5303** [NW-PaGe/.github#1]
https://github.com/NW-PaGe/.github/blob/a75fef7a7c9a8a31126be88e95b0d896a55e335d/CODE_OF_CONDUCT.md?plain=1#L61-L63

Need to add a contact to report behavior in the code of conduct. An idea is adding a group email with a defined subject line so those emails could be forwarded to doh/nw-page github moderators (pretending like those exist 😄)

` ... community leaders responsible for enforcement: [NW-PaGe Moderators](mailto:nwpagegithub@doh.wa.gov?subject=[NW-PaGe%20GitHub]%20Code%20of%20Conduct%20Report)` --> ... community leaders responsible for enforcement: [NW-PaGe Moderators](mailto:nwpagegithub@doh.wa.gov?subject=[NW-PaGe%20GitHub]%20Code%20of%20Conduct%20Report)

**Comment by DOH-FAA3303** [comment]
good idea. thoughts on adding diqa's shared mailbox for now? @DOH-PXC5303 @DOH-PNT0303
